src/core/sys/freebsd/netinet/in_.d

```diff
@@ -0,0 +1,581 @@
+//Written in the D programming language
```

Comment:
We normally put that at the top of all of the D files. AFAIK, all of the .d files in druntime and Phobos have it.